clients/iota-go/iotago/messages.go

```diff
@@ -73,6 +73,15 @@ func (p ProgrammableTransaction) PrintMoveCommand(cmdIdx int) {
 	}
 }
 
+func (p ProgrammableTransaction) DoesObjectIdExistInInputObjects(targetID *ObjectID) bool {
```

Comment:
Nitpicking, but in Go the convention is `ID` instead of `Id`:

```suggestion
func (p ProgrammableTransaction) DoesObjectIDExistInInputObjects(targetID *ObjectID) bool {
```

Also, a bit shorter: `IsObjectIDInInputObjects`

Or even: `IsInInputObjects`

Reply:
changed!